docs/index.rst

```diff
@@ -133,6 +133,12 @@ make_solo_mosaic
 | **CLI**: ``uw make_solo_mosaic -h``
 | **API**: ``import uwtools.api.make_solo_mosaic``
 
+orog_gsl
+""""""""
+
+| **CLI**: ``uw orog_gsl -h``
+| **API**: ``import uwtools.api.orog_gsl``
+
```

Comment:
There are quite a few `orog_gsl`-related diffs in this PR. I think the `notebooks` branch may need to be brought up-to-date with `main`. I'll do that shortly and we'll see if these diffs disappear.

Reply:
I merged `main` onto `notebooks`, but am still seeing a lot of diffs here that I'm sure are not part of your work. I'd like to see if merging `notebooks` onto your branch will resolve this.

But I see, at the bottom of the PR main page, that there's a conflict between the `notebooks` base branch and yours, in file `example.ipynb`. I'm not sure how this happened since it looks like only you have been working on that file. You might be able to avoid this in the future by making sure that you always create new notbook development branches based on the current `notebooks` branch -- but maybe you did, I'm not sure what happened.

If you like, and if you're sure that what's on your `test` branch is what `example.ipynb` should contain at this point, you can do the merge and resolve the conflict in one go. Here's a demo:

```
$ git checkout travis/test # I have your fork set up as a remote called 'travis'
Note: switching to 'travis/test'.
...
$ git checkout -b merge-test # I can't modify your branch, so I'll make a copy in my clone
Switched to a new branch 'merge-test'
$ git merge notebooks
Auto-merging notebooks/example.ipynb
CONFLICT (add/add): Merge conflict in notebooks/example.ipynb
Automatic merge failed; fix conflicts and then commit the result.
$ git checkout --ours -- notebooks/example.ipynb # ignore conflict and use the 'test'-branch version
$ git add notebooks/example.ipynb 
$ git status | head -n3
On branch merge-test
All conflicts fixed but you are still merging.
  (use "git commit" to conclude merge)
$ git commit
$ git status
On branch merge-test
nothing to commit, working tree clean
```
If you did this on your `test` branch, you could then `git push` to update your fork/branch and the PR.

But, before you ignore the conflict, you might like to inspect `example.ipynb` on both the `notebooks` branch and on your `test` branch to make sure there's not something in the former that you'd unintentionally be clobbering.

Let's see if we can get this branch/PR in a state where it's only showing the changes related to your latest work.

Reply:
I fixed that conflict and the PR should now only show appropriate changes. Thank you for the demo! It helped a lot.